CONTRIBUTING.md

```diff
@@ -26,27 +29,26 @@ Follow these steps to run the project on your local machine:
    ```bash
    git clone https://github.com/<Your GitHub Username>/open-bus-map-search.git
    ```
-
    Make sure to replace <Your GitHub Username> with your actual GitHub username.
 
 3. **Navigate to the Project Directory:**
-   Once the repository is cloned, navigate to the project directory by running:
-
-   ```bash
-   cd open-bus-map-search
-   ```
+    Once the repository is cloned, navigate to the project directory by running:
 
+    ```bash
+    cd open-bus-map-search
+    ```
 4. **Install Dependencies:**
-   The project uses npm to manage dependencies. Once Node is installed, you can install the project dependencies by running:
-   ```bash
-   npm install
-   ```
+   The project uses Yarn to manage dependencies. If you don't have Yarn installed, you can install it by following the instructions on the [Yarn website](https://classic.yarnpkg.com/en/docs/install). Once Yarn is installed, you can install the project dependencies by running:
+    ```bash
+    yarn
+    ```
 5. **Run the Project:**
-   After all dependencies are installed, you can start the project by running:
-   ```bash
-   npm start
-   ```
-   The project should now be running on your local machine. Open your web browser and navigate to http://localhost:3000 to view the project.
+    After all dependencies are installed, you can start the project by running:
+    ```bash
+    yarn start
+    ```
+    The project should now be running on your local machine. Open your web browser and navigate to http://localhost:3000 to view the project.
```

Comment:
seems like an accident

Reply:
Yes,Lot of things changed suddenly...
I will close this pr and open new one

Reply:
no need - you can revert the last commit and merge from main

Reply:
I see you made it work in the same PR. well done :muscle: